scripts/local-ci.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This is cool. Maybe we can even replace commands from .travis with this and avoid duplication.